packages/babel-preset-env/test/fixtures/debug/entry-corejs3-versions-decimals/stdout.txt

```diff
@@ -92,8 +92,8 @@ Using polyfills with `entry` option:
   es.array.last-index-of { "electron":"0.36", "ie":"10", "node":"6.1" }
   es.array.map { "electron":"0.36", "ie":"10", "node":"6.1" }
   es.array.of { "ie":"10" }
-  es.array.reduce { "electron":"0.36", "ie":"10" }
-  es.array.reduce-right { "electron":"0.36", "ie":"10" }
+  es.array.reduce { "chrome":"54", "electron":"0.36", "ie":"10" }
+  es.array.reduce-right { "chrome":"54", "electron":"0.36", "ie":"10" }
```

Comment:
Not a blocker, but if chrome 54 requires `es.array.reduce`. It is very likely node 6.1 requires, too. Since node 6.1 [equals](https://github.com/kangax/compat-table/blob/74122d2dc54b7d655e8e7c621dee7030b857052c/environments.json#L3302) chrome 50.

Reply:
@JLHwung if I understood correctly, this bug did not have time to get to node https://github.com/zloirock/core-js/commit/ce6673a256574c11d1ca6d4bcdba1aeb32eccc4d